From 9da506c2a3c3dc9b3e650ef3f4987581b86f8274 Mon Sep 17 00:00:00 2001
From: "Eloy Lafuente (stronk7)" <stronk7@moodle.org>
Date: Sun, 30 Dec 2012 01:07:00 +0100
Subject: [PATCH] weekly on-sync release 2.5dev

---
 version.php |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/version.php b/version.php
index c819d17..cb5b903 100644
--- a/version.php
+++ b/version.php
@@ -30,11 +30,11 @@
 defined('MOODLE_INTERNAL') || die();
 
 
-$version  = 2012120300.05;              // YYYYMMDD      = weekly release date of this DEV branch
+$version  = 2012120300.06;              // YYYYMMDD      = weekly release date of this DEV branch
                                         //         RR    = release increments - 00 in DEV branches
                                         //           .XX = incremental changes
 
-$release  = '2.5dev (Build: 20121220)'; // Human-friendly version name
+$release  = '2.5dev (Build: 20121230)'; // Human-friendly version name
 
 $branch   = '25';                       // this version's branch
 $maturity = MATURITY_ALPHA;             // this version's maturity level
-- 
1.7.9.5

